examples/CMakeLists.txt

```diff
@@ -85,5 +85,6 @@ set(albatross_example_BINARIES
   )
 foreach(TARGET IN LISTS albatross_example_BINARIES)
   target_compile_options(${TARGET} PRIVATE ${albatross_COMPILE_OPTIONS})
+  set_target_properties(${TARGET} PROPERTIES CXX_STANDARD 14)
```

Comment:
Do we want to also enforce no compiler extensions?

Reply:
Done